{"uuid":"0a25bb10-9f5e-4488-8f98-984f63aa564a","name":"Scenario Outline: The user getting error message with invalid python code for Traversal page","children":["4b50d420-2546-4aad-8852-d9f4ba721451"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"74cb10ef-b5fc-455c-bbd4-3202df7589dc-attachment","type":"image/png"},{"name":"Myscreenshot","source":"5cbf38f7-26ff-4560-a04c-8d26306202f0-attachment"}],"parameters":[],"start":1696884727520,"stop":1696884727733},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884727733,"stop":1696884727733},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884727734,"stop":1696884727734},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884727735,"stop":1696884727735}],"start":1696884727518,"stop":1696884727736}